pkgs/by-name/ra/radiance/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  libGL,
+  libGLU,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
```

The project is moving from `rec` towards `finalAttrs`
```suggestion
stdenv.mkDerivation (finalAttrs: {
```